native/src/components/OpeningHours.tsx

```diff
@@ -32,11 +36,26 @@ const TitleContainer = styled.View<{ language: string }>`
   flex-direction: ${props => contentDirection(props.language)};
 `
 
+const LinkContainer = styled.Pressable`
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+  align-items: center;
+  padding-top: 4px;
+`
+
+const Link = styled.Text`
+  font-size: 16px;
+  color: #0b57d0;
```

Comment:
please add a `textLinkColor` variable to `colors.ts` and replace it here

Reply:
Or should we even use one of our existing colors? Since the icon is already blue, I think we would not necessarily need to color the whole link blue. I don't have a personal preference though.

Reply:
I think we should color the link since that's what we've been trained to expect from browsers.